### PR TITLE
#3: create main single activity.

### PR DESCRIPTION
**Pull request by lusicom**

### CODE REVIEW

File: app/src/main/java/com/example/spacex_launchinfoapp/MainActivity.kt [hunk 1]
```diff
@@ -2,10 +2,68 @@ package com.example.spacex_launchinfoapp
...
+import androidx.drawerlayout.widget.DrawerLayout
+import com.google.android.material.navigation.NavigationView
+
+class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelectedListener {
+
+    private lateinit var drawerLayout: DrawerLayout
+    private lateinit var navigationView: NavigationView
 
-class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
```
Comment (andreysergaev) [inline app/src/main/java/com/example/spacex_launchinfoapp/MainActivity.kt:17]: please separate this method to different methods. This method is too long with a different purposes. 
You can do smth like this: 

```
override fun onCreate(savedInstanceState: Bundle?) {
       setContentView(R.layout.activity_main)
       setSupportActionBar(toolbar)
       initDrawer()
       initNavigation()
}
```

**Review by andreysergaev — CHANGES_REQUESTED** [review]
Good job, please make a changes according to my request and also structure your files using packages.

File: app/src/main/java/com/example/spacex_launchinfoapp/MainActivity.kt [hunk 2]
```diff
@@ -2,10 +2,68 @@ package com.example.spacex_launchinfoapp
...
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        val toolbar = findViewById<Toolbar>(R.id.my_toolbar)
+        setSupportActionBar(toolbar)
+
+        drawerLayout = findViewById(R.id.drawer_layout)
```
[the remark this answers is not in the document]
Comment (lusicom) [reply app/src/main/java/com/example/spacex_launchinfoapp/MainActivity.kt:24]: ✅

File: app/src/main/java/com/example/spacex_launchinfoapp/MainActivity.kt [hunk 3]
```diff
@@ -2,10 +2,68 @@ package com.example.spacex_launchinfoapp
...
+    override fun onNavigationItemSelected(item: MenuItem): Boolean {
+        when (item.itemId) {
+            R.id.nav_launches -> navigateToLaunchesScreen()
+            R.id.nav_ships -> navigateToShipsScreen()
+        }
+
+        drawerLayout.closeDrawer(GravityCompat.START)
+        return true
+    }
+
```
[the remark this answers is not in the document]
Comment (lusicom) [reply app/src/main/java/com/example/spacex_launchinfoapp/MainActivity.kt:59]: ✅

**Review by shaygini — APPROVED** [review]

**Pull request merged**